specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/ServerConnectionPolicies.json

```diff
@@ -0,0 +1,269 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/connectionPolicies/{connectionPolicyName}": {
+      "get": {
+        "tags": [
+          "ServerConnectionPolicies"
+        ],
+        "description": "Gets a server connection policy",
+        "operationId": "ServerConnectionPolicies_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "connectionPolicyName",
```

Comment:
Could reduce duplication by promoting this into a Definition and referencing it.

Reply:
we will fix it in the upcoming PR. 
all the swagger files are auto-generated. we have to fix it in SQL repo as upstream. 
given that this issue does not impact the functionality and breaking change, can we fix this issue in the next PR?